src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -260,6 +260,59 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="MaterialLeadingIconStyle"
```

Comment:
whats is the purpose of `MaterialLeadingIconStyle` && `MaterialTrailingIconStyle`?
they only make use of content, but not any of the icons? are they wip?

Reply:
They have the same purpose of [`MaterialDeleteButtonStyle`](https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L218) which is just define the [LeadingIcon](https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L218) and [TrailingIcon](https://github.com/unoplatform/Uno.Themes/blob/7c72ee3abb9de4db012c22c5bd25f859cb470e0b/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L535) appearence, because they are `Button`s.